patches/common/chromium/wtl_warnings.patch

```diff
@@ -0,0 +1,444 @@
+ 	{
+ 		T* pT = static_cast<T*>(this);
+ 		ATLASSERT(pT->m_spFileDlg.IsEqualObject(pfd));
+-		pfd;   // avoid level 4 warning
++		(void)pfd;   // avoid level 4 warning
+ 		return pT->OnFolderChanging(psiFolder);
+ 	}
+ 
+-	virtual HRESULT STDMETHODCALLTYPE IFileDialogEvents::OnFolderChange(IFileDialog* pfd)
++	virtual HRESULT STDMETHODCALLTYPE OnFolderChange(IFileDialog* pfd)
+ 	{
+ 		T* pT = static_cast<T*>(this);
+ 		ATLASSERT(pT->m_spFileDlg.IsEqualObject(pfd));
+-		pfd;   // avoid level 4 warning
++		(void)pfd;   // avoid level 4 warning
+ 		return pT->OnFolderChange();
+ 	}
+ 
+-	virtual HRESULT STDMETHODCALLTYPE IFileDialogEvents::OnSelectionChange(IFileDialog* pfd)
++	virtual HRESULT STDMETHODCALLTYPE OnSelectionChange(IFileDialog* pfd)
+ 	{
+ 		T* pT = static_cast<T*>(this);
+ 		ATLASSERT(pT->m_spFileDlg.IsEqualObject(pfd));
+-		pfd;   // avoid level 4 warning
++		(void)pfd;   // avoid level 4 warning
+ 		return pT->OnSelectionChange();
+ 	}
+ 
+-	virtual HRESULT STDMETHODCALLTYPE IFileDialogEvents::OnShareViolation(IFileDialog* pfd, IShellItem* psi, FDE_SHAREVIOLATION_RESPONSE* pResponse)
++	virtual HRESULT STDMETHODCALLTYPE OnShareViolation(IFileDialog* pfd, IShellItem* psi, FDE_SHAREVIOLATION_RESPONSE* pResponse)
+ 	{
+ 		T* pT = static_cast<T*>(this);
+ 		ATLASSERT(pT->m_spFileDlg.IsEqualObject(pfd));
+-		pfd;   // avoid level 4 warning
++		(void)pfd;   // avoid level 4 warning
+ 		return pT->OnShareViolation(psi, pResponse);
+ 	}
+ 
+-	virtual HRESULT STDMETHODCALLTYPE IFileDialogEvents::OnTypeChange(IFileDialog* pfd)
++	virtual HRESULT STDMETHODCALLTYPE OnTypeChange(IFileDialog* pfd)
+ 	{
+ 		T* pT = static_cast<T*>(this);
+ 		ATLASSERT(pT->m_spFileDlg.IsEqualObject(pfd));
+-		pfd;   // avoid level 4 warning
++		(void)pfd;   // avoid level 4 warning
+ 		return pT->OnTypeChange();
+ 	}
+ 
+-	virtual HRESULT STDMETHODCALLTYPE IFileDialogEvents::OnOverwrite(IFileDialog* pfd, IShellItem* psi, FDE_OVERWRITE_RESPONSE* pResponse)
++	virtual HRESULT STDMETHODCALLTYPE OnOverwrite(IFileDialog* pfd, IShellItem* psi, FDE_OVERWRITE_RESPONSE* pResponse)
+ 	{
+ 		T* pT = static_cast<T*>(this);
+ 		ATLASSERT(pT->m_spFileDlg.IsEqualObject(pfd));
+-		pfd;   // avoid level 4 warning
++		(void)pfd;   // avoid level 4 warning
+ 		return pT->OnOverwrite(psi, pResponse);
+ 	}
+ 
+@@ -1222,7 +1222,7 @@ public:
+ 		HRESULT hRet = m_spFileDlg.CoCreateInstance(CLSID_FileOpenDialog);
+ 
+ 		if(SUCCEEDED(hRet))
+-			_Init(lpszFileName, dwOptions, lpszDefExt, arrFilterSpec, uFilterSpecCount);
++			this->_Init(lpszFileName, dwOptions, lpszDefExt, arrFilterSpec, uFilterSpecCount);
+ 	}
+ 
+ 	IFileOpenDialog* GetPtr()
+@@ -1244,6 +1244,7 @@ public:
+ 	                     const COMDLG_FILTERSPEC* arrFilterSpec = NULL, 
+ 	                     UINT uFilterSpecCount = 0U) : CShellFileOpenDialogImpl<CShellFileOpenDialog>(lpszFileName, dwOptions, lpszDefExt, arrFilterSpec, uFilterSpecCount)
+ 	{ }
++	virtual ~CShellFileOpenDialog() = default;
```

Comment:
Fixes a chromium-style issue (though that's weird, I thought chromium-style wasn't supposed to apply to things in `third_party/`...)